app/views/mailer/mfa_required_soon_announcement.text.erb

```diff
@@ -0,0 +1,23 @@
+👋 Hi <%= @user.handle %>,
+
+Recently, we've announced our security-focused ambitions to the community. Attacks on the software supply chain are increasing and the second most common attack is account takeovers.
+
+Next week, on August 15, 2022, we will begin requiring maintainers like yourself, who are owners of packages with more than 180 million downloads to have multi-factor authentication (MFA) enabled.
+
+If you don't have MFA enabled by this date, a number of operations will be restricted.
+These restrictions include privileged operations (ie. push, yank, add/remove owners) (https://guides.rubygems.org/mfa-requirement-opt-in/#privileged-operations).
```

Comment:
Perhaps we can be more explicit in this email on what we are limiting so people that read this email knows what to expect and wouldn't be surprised (maybe like what we are saying in the [blog post?](https://github.com/rubygems/rubygems.github.io/pull/121/files#diff-e93070038fbc5d2430174deaa0c86f9d05990d9b8ead73ae6e0dbd83d323eeb4R13-R14)).

Reply:
I like where your head's at 💯. I toiled over the messaging approach of this email:
* More thorough, but lengthy
* More concise, but may lack all the details

Ultimately, I was aiming for the latter option. Since, this set of users would have in theory received the first mailer we sent out. It was thorough but lengthy. I was hoping to take a different approach this time to provide _enough_ information but be a bit more concise to focus on the key message of "please action on this asap".

If anything, I'd be game to trim down more copy if possible. What do you think?

Reply:
That makes sense, however I'm concerned that we are omitting important details. Maybe someone would expect to still be able to sign in on the command line but the email only says gem push/yank, owner add/remove.

I agree we should make the copy as concise as possible though. Reading over it again, here are potential parts I find that can be omitted/trimmed.

1. `Recently, we've announced our security-focused ambitions to the community.` we can link the blog post in this sentence instead of having the `For more information...` sentence at the end on the email.
2. `Attacks on the software supply chain are increasing and the second most common attack is account takeovers.` can be omitted since they can read the announcement.
3. `We recommend setting the ... [mfa level]` could be deleted as well since users [cannot change their level to UI only](https://github.com/rubygems/rubygems.org/pull/3084) anymore and the default is `ui and api`.

Reply:
Ooo I like that finessing 💯. I'll take it for a spin!

Reply:
Done! Trimmed down the areas you suggested which made room to list the restrictions in full!